render/detailed/connections.go

```diff
@@ -206,10 +211,15 @@ func connectionRows(r report.Report, in map[connection]int, includeLocal bool) [
 			connection.Linkable = false
 		}
 		if includeLocal {
+			// Does localNode have a DNS record in it?
+			label := row.localAddr
+			if set, ok := row.localNode.Sets.Lookup(endpoint.ReverseDNSNames); ok && len(set) > 0 {
+				label = set[0]
+			}
```

Comment:
Just picking the first one seems a bit random. any better heuristics we could use? Shortest? Longest?